api/__tests__/all.test.js

```diff
@@ -218,33 +241,98 @@ describe("POST /api/articles/:article_id/comments", () => {
         expect(response.body.msg).toEqual("Username not found");
       });
   });
-  test("Extra properties in post object are ignored", () => {
+
+  test("if username not found in the database return correct error", () => {
     return request(app)
       .post("/api/articles/4/comments")
-      .send({
-        username: "lurker",
-        body: "weird article",
-        extraProp: "extraValue",
-      })
-      .expect(201)
+      .send({ username: "unknownUser", body: "good article" })
+      .expect(404)
       .then((response) => {
-        const commentObject = response.body.comment;
-        const expectedCommentObject = {
-          author: "lurker",
-          body: "weird article",
+        expect(response.body.msg).toEqual("Username not found");
+      });
+  });
+});
+
+//PATCH TESTS
+describe("PATCH /api/articles/:article_id", () => {
+  test("patch a article using the article_id, return an object with the key of inc_votes and the value of how many votes to be increased or decreased by then returning updated article", () => {
```

Comment:
It would be good to combine the following tests a little bit if possible, we have some duplicate assertions at the moment. Lets just have one test for incrementing votes and assert for everything the endpoint responds with. We could do a second for decrementing too as you have, but again testing for the entire response would be good.